.deps/prod.md

```diff
@@ -216,7 +216,7 @@
 | [`performance-now@2.1.0`](git://github.com/braveg1rl/performance-now.git) | MIT | [CQ16925](https://dev.eclipse.org/ipzilla/show_bug.cgi?id=16925) |
 | [`popper.js@1.16.1`](git+https://github.com/FezVrasta/popper.js.git) | MIT | [CQ22353](https://dev.eclipse.org/ipzilla/show_bug.cgi?id=22353) |
 | [`postcss@8.2.4`](https://github.com/postcss/postcss.git) | MIT | clearlydefined |
-| [`prettier@2.2.1`](https://github.com/prettier/prettier.git) | MIT | [CQ23020](https://dev.eclipse.org/ipzilla/show_bug.cgi?id=23020) |
+| [`prettier@2.0.5`](https://github.com/prettier/prettier.git) | MIT | clearlydefined |
```

Comment:
As I remember, it was a  vulnerability and we fixed it in https://github.com/eclipse/che-dashboard/pull/150/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R153
Is not it?

Reply:
The list of vulnerable libraries from the PR:

> axios to version 0.21.1
> websocket-extensions to version 0.1.4
> node-forge to version 0.10.0
> elliptic to version 6.5.3

`prettier` wasn't on that list.